spec/models/activity_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Activity, type: :model do
+  describe 'validations' do
+    let(:subject) do
+      build_stubbed(:activity)
+    end
+
+    it 'is valid with valid parameters' do
+      expect(subject).to be_valid
+    end
+
+    it 'validates presence of owner' do
+      subject.owner = nil
+      subject.validate
+      expect(subject.errors[:owner]).to_not be_empty
+    end
```

Comment:
We have used the `shoulda-matchers` to do this sort of validation checking in other projects.